src/modules/Header/components/Header.js

```diff
@@ -31,6 +30,7 @@ const HeaderBarUpper = styled.div`
   max-width: 100%;
   background: ${COLORS.WHITE};
   max-height: 50px;
+  padding: 20px 0 12px 0;
```

Comment:
It would be better to define paddings in em or rem units. Pixels don't scale. Tailwind provides small predefined classes for it.